daisy_workflows/export/disk_export.wf.json

```diff
@@ -47,7 +47,6 @@
         {
           "Name": "disk-${NAME}",
           "SourceImage": "${export_instance_disk_image}",
-          "SizeGb": "${export_instance_disk_size}",
```

Comment:
What's the motivation for removing this?

Reply:
This is the OS disk for raw image export. It doesn't have to be set to 200GB (the default) or more if we pass the value based on image size. We don't store any data on it when doing raw file export, it's all written directly to GCS

Reply:
:+1: